libraries/AP_Airspeed/AP_Airspeed_MS4525.cpp

```diff
@@ -47,6 +47,7 @@ bool AP_Airspeed_MS4525::init()
         uint8_t bus;
         uint8_t addr;
     } addresses[] = {
+        { 2, MS4525D0_I2C_ADDR },
```

Comment:
the bus number is actually something that should be coming from the board.  The I2C addr is ok to leave on the sensor since it depends more on the sensor than on the board.